client/components/theme/more-button.jsx

```diff
@@ -121,7 +121,7 @@ class ThemeMoreButton extends Component {
 
 ThemeMoreButton.propTypes = {
 	// See Theme component propTypes
-	theme: PropTypes.object,
+	themeId: PropTypes.number,
```

Comment:
`themeId`s are actually `string`s (like `radcliffe-2` or `twentysixteen`), not `number`s.

Reply:
and that's why you should never ever put your devtools' console in `info only` :( sorry about that.